libs/middleware/core/include/middleware/core/types.h

```diff
@@ -0,0 +1,77 @@
+// Copyright 2025 BMW AG
+
+#pragma once
+
+#include <etl/limits.h>
+#include <etl/platform.h>
+
+#include <cstdint>
+
+namespace middleware
+{
+namespace core
+{
+
+using ServiceId = uint16_t;
```

Comment:
Why those type aliases?
They provide no type safety while obfuscating the readability of the code.